src/api/app/controllers/source_controller.rb

```diff
@@ -697,7 +697,7 @@ def update_package_meta
       end
 
       if pkg and not pkg.disabled_for?('sourceaccess', nil, nil)
-        if FlagHelper.xml_disabled_for?(rdata, 'sourceaccess')
+        if FlagHelper.xml_disabled_for?(rdata, 'sourceaccess') and not User.current.is_admin?
```

Comment:
Please do not use ``and``! Use ``&&`` instead!
http://phrogz.net/ProgrammingRuby/language.html#table_18.4